src/index.js

```diff
@@ -64,7 +64,10 @@ export default class ScrollBooster {
             textSelection: false,
             inputsFocus: true,
             emulateScroll: false,
+            preventDefaultOnEmulateScroll: false, // 'vertical', 'horizontal'
             pointerDownPreventDefault: true,
+            lockScrollOnDragDirection: false, // 'vertical', 'horizontal'
+            DragDirectionTolerance: 40,
```

Comment:
can you please make this prop name to start with lowercase letter, for consistency with other names?

Reply:
Oops, missed that one 👍